### DISCUSSION TITLE
GitHub CoPilot transfer access

### DISCUSSION THREAD

**Discussion by Ctouchdev** [community/community#16487] [Copilot Conversations]
Hi,

I have recently accepted a position at a new company. While working for American Airlines, I did have access to CoPilot. Is there any way to transfer my access to this email? (My personal account)

**Comment by Ctouchdev** [comment]
Bump

**Comment by dailyainick** [comment]
I too would like to be able to transfer "ownership" of my copilot subscription to a different github account . There is another ongoing discussion with the same request [here](https://github.com/orgs/community/discussions/9149#discussion-3757734)
